FOUNDATIONS.md

```diff
@@ -6,10 +6,14 @@ ensure the success of future improvements (especially type systems).
 **Definitions**
 
 * Block: A block is a chunk of an IPLD DAG, encoded in a format. Blocks have CIDs.
-* Fragment: A piece of an IPLD DAG. Blocks contain fragments.
 * Node: A node is a *point* in an IPLD DAG (object, array, number, etc.).
-* Link: A link is an IPLD Node that points to another IPLD Node.
-* Path: A paths a human readable pointer to an IPLD Node.
+  Many nodes can exist encoded inside one Block.
+* Link: A link is a kind of IPLD Node that points to another IPLD Node.
```

Comment:
For the sake of the unfamiliar reader, would it be ok to add reference to CID in here? We do have the spec in this repo after all.

> "Currently an Link is represented as a [CID](https://github.com/ipld/specs/blob/master/block-layer/CID.md) although additional Link representations may be introduced in future."

Reply:
Hm.  You know, on further thought, I almost wanted to push back on this, and say "the concept of a link should be clear without referring to the implementation details of CIDs".

But we already have the phrase "blocks have CIDs" just a few lines above, so... might as well be consistent about it, in that case.